packages/button/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@rainbow-me/button",
+  "description": "Rainbow connector button",
+  "version": "0.1.0",
+  "files": [
+    "dist"
+  ],
+  "keywords": [],
+  "author": "Rainbow",
+  "license": "MIT",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js",
+    "./styles.css": "./node_modules/@rainbow-me/rainbowkit/styles.css"
```

Comment:
Because `rainbowkit` is a peer dependency, this might be an issue. I think we should bundle this with `/dist` in the `built.js` script; that should always work at build-time because rainbowkit is a dev dependency